readthedocs/api/v2/serializers.py

```diff
@@ -46,9 +52,14 @@ class ProjectAdminSerializer(ProjectSerializer):
         slug_field='feature_id',
     )
 
+    environment_variables = AllEnvVarsField()
```

Comment:
I think this can be `serializers.SerializerMethodField()`

Reply:
Ah, so that’s what `get_environment_variables` is for! Since we don’t want to change the V2 API’s behavior (right?) that means reverting the change to that method.

Reply:
We need to change it, since it's used by the builders to filter no-public vars. It's a breaking change, but this is used internally only, and hasn't been documented yet, so I think it should be fine to change it. /cc @humitos

Reply:
I'm not sure I follow. Are we just adding `public` field in the API response? If that's the case, that's not a breaking change.

Reply:
Oh, I just saw that we are changing the structure from:

```js
{
 name: value
}
```

to 

```js
{
 name: {
  value: str
  public: bool
 } 
}
```

However, as @stsewd said, I don't think it's super important in this case since it's used internally only.